Ferlab-Ste-Justine/ferlab-ui
# SKFP 65 combine queries

# FEATURE Combine Queries

- closes #SKFP-65

## QA

Storybook: https://oliviercperrier.github.io/ferlab-ui/?path=/story/ferlab-components-querybuilder-querybuilder--query-builder-multi-query-with-combine 

1. Select multiple queries
2. Click the combine button (you can also change the combine operator by clicking the 3 dots)
-> A new query is created with the color matching the references

3. Inside the new query, remove the reference to other queries
-> Reference are removed correctly

4. Create a new combined query
5. Delete a query that has a reference inside the combined query
-> Reference(s) are remove everywhere, query index are updated

6. Create a new combined query
7. Duplicate it
-> Combined query is correctly duplicated

8. Combine button should only be displayed when 2 or more query are selected.
-> When the combine button is displayed, everything else in the action bar is hidden.

## Mention

@francisl
